main.mo

```diff
@@ -171,6 +171,8 @@ shared({caller}) actor class Rex(
         caller : Principal,
         revert : () -> ()
     ) : async Result.Result<(), Interface.MintError> {
+        if (amount <= 10_000) return #ok;
```

Comment:
Amount could be 0 here I think. It's nice to be able to set price to 1 e8s for testing

Reply:
Since we deduct the fee from the amount, the amount should be bigger than `10_000` right?

Reply:
Hmm, yeah... so long as we are including tx fee in the mint price, mint prices below `10_000` are impossible. This makes sense. Nice!